tests/test_sphinx_autodoc_typehints.py

```diff
@@ -58,6 +58,8 @@
 S = TypeVar("S", bound="miss")  # type: ignore # miss not defined on purpose # noqa: F821
 W = NewType("W", str)
 P = typing_extensions.ParamSpec("P")
+P_args = P.args
+P_kwargs = P.kwargs
```

Comment:
MyPy fails with 
```
tests/test_sphinx_autodoc_typehints.py:61: error: "object" has no attribute "args"  [attr-defined]
tests/test_sphinx_autodoc_typehints.py:62: error: "object" has no attribute "kwargs"  [attr-defined]
```

Reply:
Add `  # type:ignore[attr-defined]` after each of these (two spaces before # one after it).